src/main/java/uk/gov/hmcts/reform/unspec/stateflow/utils/PredicateUtils.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.unspec.stateflow.utils;
+
+import uk.gov.hmcts.reform.unspec.model.CaseData;
+
+import java.util.function.Predicate;
+
+import static uk.gov.hmcts.reform.unspec.enums.CaseState.STAYED;
+
+public class PredicateUtils {
+
+    public static final Predicate<CaseData> claimantIssueClaim = caseData ->
+        caseData.getClaimIssuedDate() != null
+            && caseData.getLegacyCaseReference() != null;
```

Comment:
Wonder if we should stick to the bare minimum and only rely on the most simple data as marker. In this scenario would ClaimIssuedDate not be enough?

Reply:
Done in the next PR.